Kickstarter-iOS/Features/PledgeView/Controllers/PostCampaignCheckoutViewController.swift

```diff
@@ -204,6 +205,13 @@ final class PostCampaignCheckoutViewController: UIViewController {
       }
 
     self.paymentMethodsViewController.view.rac.hidden = self.viewModel.outputs.paymentMethodsViewHidden
+
+    self.viewModel.outputs.configureStripeIntegration
+      .observeForUI()
+      .observeValues { merchantIdentifier, publishableKey in
+        STPAPIClient.shared.publishableKey = publishableKey
```

Comment:
Thanks @scottkicks! Just copied this directly over from `PledgeViewModel/PledgeViewController`.

Reply:
It's not clear to me why we only set the Stripe keys if the payment view isn't hidden - does Stripe charge us for setup? Seems odd, but I'm not willing to plumb those depths right now.